docs/BOOTING.md

```diff
@@ -127,6 +127,7 @@ in grub.cfg with graphical GRUB menu to get the device to boot again.
     10. `eve_install_skip_persist` - do not install persist partition onto device. May be selected from graphical GRUB menu.
     11. `eve_install_skip_rootfs` - do not install rootfs partition onto device. May be selected from graphical GRUB menu.
     12. `eve_install_skip_zfs_checks` - install zfs by skipping minimum requirement checks.
+    13. `eve_install_zfs_with_raid_level` - Sets raid level for zfs storage. Valid values are none,raid1,raid5,raid6. Default value is none.
```

Comment:
My earlier question seems to have been missed so after invoking grep I'll make it into a statement.
We currently do not use raid5 nor raid6 in the API nor in the EVE code, but we do use raid1, raidz1, raidz2.

It doesn't make sense introducing raid5 and raid6 in the installer; we should at least use consistent terms.

Reply:
Actually we display None, RAID1, RAID5 and RAID6 on UI today. We made that decision as per your suggestion in one of our storage team calls because raidz2 and raidz1 are zfs specific and we wanted to display industry standard terms, that also helps in future if we decide to use some other filesystem. That's why I made installer to be in sync with what user sees in UI.

Reply:
But that's already inconsistent because the API calls them raidz1 raidz2. 
Please make it consistent one way or another.

Note that we can change the enum strings in the API without any wire compatibility issues; the protobuf wire encoding just uses the tags and the types.
So if we want to use Raid5,6,7 then change the API and EVE to use those and not just the UI. Also the zmsg API in zedcloud should match.

Reply:
Working on that.